packages/ia-components/sandbox/v2mocks/details-page-container.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import { Item } from 'ia-js-client'
+// import { Item } from 'ia-js-client'
+
+import { Item } from '../../../ia-js-client'
```

Comment:
use the traditional import statement. then use `yarn run lerna link` to locally symlink the packages.